portfolio/src/main/webapp/script.js

```diff
@@ -41,6 +41,17 @@ async function htmlInject(templatePath, targetID) {
   }
 }
 
+/**
+ * Updates the comment section after a new comment is submitted
+ */
+function updateCommentSection() {
+  // Wait for the form to submit before resetting
+  setTimeout(function(){ 
+    document.getElementById("comment-form").reset();
+    getComments();
+    }, 1000);
```

Comment:
I think with this design, if your webserver or internet connection is slow, `getComments()` might be called before the `/new-comment` post is finished, which might mean that you won't get back your newly added comment.

A better approach might be to get rid of the HTML form tag, and instead add an 'onclick' handler to the submit button.  That could then use the fetch API to post to `/new-comment` and in the .then function it could clear the form and call `getComments()`.  Let me know if this makes sense or not

Reply:
+1 for using `.then()`.

Reply:
Makes sense. I'll add that now!

Reply:
It took some research to find the right imports, but I think I've done it correctly!